Plugins/Flow.Launcher.Plugin.WebSearch/SuggestionSources/Baidu.cs

```diff
@@ -23,7 +23,7 @@ public override async Task<List<string>> SuggestionsAsync(string query, Cancella
             try
             {
                 const string api = "http://suggestion.baidu.com/su?json=1&wd=";
-                result = await Http.GetAsync(api + Uri.EscapeUriString(query), token).ConfigureAwait(false);
+                result = await Http.GetAsync(api + Uri.EscapeDataString(query), token).ConfigureAwait(false);
```

Comment:
what's the difference between these two?

Reply:
`EscapeUriString` is deprecated: https://docs.microsoft.com/en-us/dotnet/api/system.uri.escapeuristring?view=net-6.0